src/koop/config/default.json

```diff
@@ -1,22 +1,5 @@
 {
   "logger" : {
     "level" : "debug"
-  },
-  "port" : @KOOP_PORT@,
```

Comment:
I needed to change the default.json and remove the tokenized property values.  These are now in the dev.json configuration file.